solo-chains/runtime/dancelight/src/lib.rs

```diff
@@ -1242,13 +1238,20 @@ impl pallet_external_validators::Config for Runtime {
     type ValidatorRegistration = Session;
     type UnixTime = Timestamp;
     type SessionsPerEra = SessionsPerEra;
-    type OnEraStart = ExternalValidatorSlashes;
+    type OnEraStart = (ExternalValidatorSlashes, ExternalValidatorsRewards);
     type OnEraEnd = ();
     type WeightInfo = weights::pallet_external_validators::SubstrateWeight<Runtime>;
     #[cfg(feature = "runtime-benchmarks")]
     type Currency = Balances;
 }
 
+impl pallet_external_validators_rewards::Config for Runtime {
+    type EraIndexProvider = ExternalValidators;
+    type HistoryDepth = ConstU32<64>;
```

Comment:
There's a different HistoryDepth in `impl pallet_external_validators::Config for Runtime`, shall we unify them into a constant? Or do we want them to be different?

Reply:
I think there is also a difference in polkadot from what I understand, if so, it's ok if we maintain them different